testutil/keeper/unit_test_helpers.go

```diff
@@ -140,51 +140,56 @@ func NewInMemConsumerKeeper(params InMemKeeperParams, mocks MockedKeepers) consu
 	)
 }
```

Comment:
Changes to this file are syntactical and don't change anything functionally. A params struct is now passed into `GetProviderKeeperAndCtx`, where those parameters can be mutated in anyway before keeper instantiation